experimental/traffic-portal/src/app/shared/generic-table/generic-table.component.html

```diff
@@ -49,7 +50,7 @@
 	<ul>
 		<li *ngFor="let item of contextMenuItems" role="menuitem">
 			<ng-container *ngIf="!isAction(item)">
-				<a *ngIf="!isDisabled(item)" [href]="href(item)" [target]="item.newTab ? '_blank' : '_self'">{{item.name}}</a>
+				<a *ngIf="!isDisabled(item)" [href]="href(item)" (click)="clickContextMenu($event, item)" [target]="item.newTab ? '_blank' : '_self'">{{item.name}}</a>
```

Comment:
anchors should not use click event handlers. the way that handler works, the browser doesn't handle navigation, which essentially renders the `href` attribute useless.

Reply:
It still can use the default browser behavior. Regardless, I know of a better & easier way of this (way over-engineered this one).